src/python/grapl_analyzerlib/grapl_analyzerlib/nodes/base.py

```diff
@@ -0,0 +1,212 @@
+import json
+from typing import Any, TypeVar, Set, Type, Optional
+
+from grapl_analyzerlib.grapl_client import GraphClient
+from grapl_analyzerlib.node_types import (
+    PropType,
+    PropPrimitive,
+)
+from grapl_analyzerlib.queryable import Queryable
+from grapl_analyzerlib.schema import Schema
+from grapl_analyzerlib.viewable import Viewable
+
+BCQ = TypeVar("BCQ", bound="BaseQuery")
+BCV = TypeVar("BCV", bound="BaseView")
+
+
+class BaseSchema(Schema):
+    def __init__(self, properties=None, edges=None, view=None):
```

Comment:
could you type properties, edges, view if it makes sense to do so

Reply:
Yep, for sure.